04/dbernhard/tests_if4.oat

```diff
@@ -0,0 +1,14 @@
+int program(int argc, string[] args) {
+    return 2*function(0) + function(1) + function(2);
+}
+
+int function(int x) {
+    if (x == 0) {
+        return 1;
+    } else if (x == 1) {
+        return 0;
```

Comment:
Would it be better to return a different number here? Something like 2, so from the result we can verify that at each call we take a different branch of the nested if then else